<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1128</width>
    <height>780</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Particles editor</string>
  </property>
  <property name="locale">
   <locale language="English" country="UnitedStates"/>
  </property>
  <widget class="QWidget" name="centralwidget"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1128</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="actionUndo"/>
    <addaction name="actionRedo"/>
    <addaction name="separator"/>
    <addaction name="actionDelete"/>
    <addaction name="separator"/>
    <addaction name="actionAdd_frame"/>
    <addaction name="actionAdd_emitter"/>
    <addaction name="actionAdd_visibility_control"/>
    <addaction name="actionAdd_gravity"/>
    <addaction name="actionAdd_blocker"/>
    <addaction name="actionAdd_heater"/>
    <addaction name="actionAdd_gas_source"/>
    <addaction name="separator"/>
    <addaction name="actionAdd_animation_from_fbx"/>
   </widget>
   <widget class="QMenu" name="menuManipulator">
    <property name="title">
     <string>Manipulator</string>
    </property>
    <addaction name="actionSetMoveManipulator"/>
    <addaction name="actionSetRotationManipulator"/>
    <addaction name="actionSetScaleManipulator"/>
    <addaction name="separator"/>
    <addaction name="actionSetLocalOrientation"/>
    <addaction name="actionSetGlobalOrientation"/>
   </widget>
   <widget class="QMenu" name="menuStatistic">
    <property name="title">
     <string>Statistic</string>
    </property>
    <addaction name="actionStatistic"/>
   </widget>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionLoad"/>
    <addaction name="separator"/>
    <addaction name="actionSave"/>
    <addaction name="actionSave_as"/>
    <addaction name="separator"/>
    <addaction name="actionLoad_environment"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuManipulator"/>
   <addaction name="menuStatistic"/>
  </widget>
  <action name="actionAdd_frame">
   <property name="text">
    <string>Add frame</string>
   </property>
  </action>
  <action name="actionUndo">
   <property name="text">
    <string>Undo</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Z</string>
   </property>
  </action>
  <action name="actionRedo">
   <property name="text">
    <string>Redo</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Y</string>
   </property>
  </action>
  <action name="actionDelete">
   <property name="text">
    <string>Delete</string>
   </property>
   <property name="shortcut">
    <string>Del</string>
   </property>
  </action>
  <action name="actionAdd_animation_from_fbx">
   <property name="text">
    <string>Add animation from fbx</string>
   </property>
  </action>
  <action name="actionSetMoveManipulator">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Moving</string>
   </property>
   <property name="shortcut">
    <string>G</string>
   </property>
  </action>
  <action name="actionSetRotationManipulator">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Rotation</string>
   </property>
   <property name="shortcut">
    <string>R</string>
   </property>
  </action>
  <action name="actionSetLocalOrientation">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Local orientation</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+L</string>
   </property>
  </action>
  <action name="actionSetGlobalOrientation">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Global orientation</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+G</string>
   </property>
  </action>
  <action name="actionAdd_LOD">
   <property name="text">
    <string>Add LOD</string>
   </property>
  </action>
  <action name="actionAdd_direct_light">
   <property name="text">
    <string>Add direct light</string>
   </property>
  </action>
  <action name="actionStatistic">
   <property name="text">
    <string>Statistic window</string>
   </property>
  </action>
  <action name="actionAdd_ambient_light">
   <property name="text">
    <string>Add ambient light</string>
   </property>
  </action>
  <action name="actionSetScaleManipulator">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Scale</string>
   </property>
   <property name="shortcut">
    <string>S</string>
   </property>
  </action>
  <action name="actionAdd_model_from_Blender">
   <property name="text">
    <string>Add model from Blender's fbx</string>
   </property>
  </action>
  <action name="actionAdd_model_from_3DMax">
   <property name="text">
    <string>Add model from 3DMax's fbx</string>
   </property>
  </action>
  <action name="actionAdd_model_from_obj">
   <property name="text">
    <string>Add model from obj</string>
   </property>
  </action>
  <action name="actionAdd_material">
   <property name="text">
    <string>Add material</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>Save</string>
   </property>
  </action>
  <action name="actionSave_as">
   <property name="text">
    <string>Save as ...</string>
   </property>
  </action>
  <action name="actionLoad">
   <property name="text">
    <string>Load</string>
   </property>
  </action>
  <action name="actionLoad_environment">
   <property name="text">
    <string>Load environment</string>
   </property>
  </action>
  <action name="actionSave_environment">
   <property name="text">
    <string>Save environment</string>
   </property>
  </action>
  <action name="actionSave_environment_as">
   <property name="text">
    <string>Save environment as ...</string>
   </property>
  </action>
  <action name="actionAdd_environment_model">
   <property name="text">
    <string>Add model to environment</string>
   </property>
  </action>
  <action name="actionAdd_emitter">
   <property name="text">
    <string>Add emitter</string>
   </property>
  </action>
  <action name="actionAdd_animation">
   <property name="text">
    <string>Add animation</string>
   </property>
  </action>
  <action name="actionAdd_visibility_control">
   <property name="text">
    <string>Add visibility control</string>
   </property>
  </action>
  <action name="actionAdd_gravity">
   <property name="text">
    <string>Add gravity</string>
   </property>
  </action>
  <action name="actionAdd_blocker">
   <property name="text">
    <string>Add blocker</string>
   </property>
  </action>
  <action name="actionAdd_heater">
   <property name="text">
    <string>Add heater</string>
   </property>
  </action>
  <action name="actionAdd_gas_source">
   <property name="text">
    <string>Add gas source</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
